docs/plugins.md

```diff
@@ -387,7 +387,6 @@ const step = codeceptjs.container.plugins('commentStep');
 const Given = () => step`Given`;
 const When = () => step`When`;
 const Then = () => step`Then`;
-```
 
 Scenario('project update test', async (I) => {
```

Comment:
Would you mind updating this as well? This shall be {I}

Reply:
Thanks @PeterNgTr!
I fixed it according to your advice. And I found one equally wrong place and fixed it as well.